.travis.yml

```diff
@@ -44,24 +56,19 @@ after_success:
     fi
 
 before_deploy:
-  - go get github.com/inconshreveable/mousetrap
-  - go get github.com/mattn/go-isatty
-  - export build_file_name=wsk
-  - export zip_file_name=OpenWhisk_CLI
-  - chmod +x tools/travis/build_tag_releases.sh
-  - ./tools/travis/build_tag_releases.sh $build_file_name $zip_file_name
-  - export RELEASE_PKG_FILE=$(ls $zip_file_name-*.*)
-  - echo "Deploying $RELEASE_PKG_FILE to GitHub releases."
+  - echo "Deploying $(ls ./release/${zip_file_name}-*.tgz ./release/${zip_file_name}-*.zip) to GitHub releases."
```

Comment:
`echo "Deploying $(ls ./release/${zip_file_name}-*.tgz, and ./release/${zip_file_name}-*.zip) to GitHub releases."`

Reply:
Not going to give the result you expect.  The shell expansion is creating a list of all filenames so the , and is out of place.  Let's see if this format is intolerable -- we can get more clever if it is.